packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -119,8 +130,30 @@ const hasValidParticipant = (chatMessage: ChatMessageWithStatus): boolean =>
  * @public
  */
 export const messageThreadSelector: MessageThreadSelector = createSelector(
-  [getUserId, getChatMessages, getLatestReadTime, getIsLargeGroup],
-  (userId, chatMessages, latestReadTime, isLargeGroup) => {
+  [getUserId, getChatMessages, getLatestReadTime, getIsLargeGroup, getReadReceipts, getParticipants],
+  (userId, chatMessages, latestReadTime, isLargeGroup, readReceipts, participants) => {
+    // get number of participants
+    const messageThreadParticipantCount = Object.values(participants).length - 1;
```

Comment:
So this is the participantCount but not includes myself, I am wondering if we could give this a better name to indicate this:
When I am looking at this name, I was expecting it includes myself. Could be 2 solutions - 
1. give it a new name
2. not to -1 for this value, but -1 in messageThread component when rendering it